### ISSUE TITLE
Cannot find module /src/iconVersion

### ISSUE THREAD

**Issue by B-E-A-F** [AlaskaAirlines/auro-select#196]
## Describe the bug

While attempting to use `svelte-kit`, I'm unable to use `auro-select`. I think it could possibly be an SSR issue?
```bash
[vite] Error when evaluating SSR module /src/routes/contactdetails/+page.svelte: failed to import "@aurodesignsystem/auro-select"
|- Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/Michael.Archer@alaskaair.com/Documents/Projects/ManageContactDetails/node_modules/@aurodesignsystem/auro-menu/src/iconVersion' imported from /Users/Michael.Archer@alaskaair.com/Documents/Projects/ManageContactDetails/node_modules/@aurodesignsystem/auro-menu/src/auro-menuoption.js
    at finalizeResolution (node:internal/modules/esm/resolve:264:11)
    at moduleResolve (node:internal/modules/esm/resolve:917:10)
    at defaultResolve (node:internal/modules/esm/resolve:1130:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36)
```

## To Reproduce

1. Create a `svelte-kit` app
1. `npm i @aurodesignsystem/auro-menu`
1. Import component `import '@aurodesignsystem/auro-select'`
1. See error

## Screenshots

<img width="1096" alt="image" src="https://github.com/AlaskaAirlines/auro-select/assets/20789620/3389a74f-af73-4b60-b621-36aa5990a167">

## Desktop (please complete the following information):

 - OS: MacOS (m1)
 - Browser: Chrome
 - Version: 126.0.6478.127

**Comment by Patrick-Daly-AA** [comment]
first glance looks like this might be an issue with either the build process or the bundler.  @jason-capsule42 to do further investigation.

**Comment by ArcherScript** [comment]
Hi @jason-capsule42, is there an update on this? I'm not blocked yet, but very close to being a blocker on using auro for our current project.

**Comment by blackfalcon** [comment]
The auro-select component has not been updated to be fully ESM SSR compatible, please see https://github.com/orgs/AlaskaAirlines/discussions/502

The auro-select element has a dependency on auro-menu which has a dependency on mark.js which hasn't had a release for [5 years](https://www.npmjs.com/package/mark-js?activeTab=versions). 

Unless we can find a ESM workaround for mark.js, please see this recommendation: 
https://github.com/orgs/AlaskaAirlines/discussions/502#discussioncomment-8456578

**Comment by blackfalcon** [comment]
Your error stack does elude to this line of code: 

https://github.com/AlaskaAirlines/auro-menu/blob/main/src/auro-menuoption.js#L13

To be ESM compatible, this JavaScript reference needs to use `.js`, `.cjs` or `.mjs`. That could also be part of the issue.

**Comment by jordanjones243** [comment]
This was fixed by auro-menu v4.0.0 release and therefore this ticket can be closed.
